wagtail/tests/test_management_commands.py

```diff
@@ -697,6 +698,40 @@ def test_purge_revisions_with_date_cutoff(self):
         )
 
 
+class TestPurgeEmbedsCommand(TestCase):
+    fixtures = ["test.json"]
+
+    def setUp(self):
+        # create dummy Embed objects
+        for i in range(5):
+            embed = Embed(
+                hash=f"{i}",
+                url="https://www.youtube.com/watch?v=Js8dIRxwSRY",
+                max_width=None,
+                type="video",
+                html="test html",
+                title="test title",
+                author_name="test author name",
+                provider_name="test provider name",
+                thumbnail_url="http://test/thumbnail.url",
+                width=1000,
+                height=1000,
+            )
+            embed.save()
+
+    def test_purge_embeds(self):
+        # fetch all dummy embeds
```

Comment:
```suggestion
        """ 
        fetch all dummy embeds and confirm they are deleted when the management command runs
        """
```

Use pydoc strings here to better describe the test.